nuxt.config.js

```diff
@@ -17,7 +23,7 @@ module.exports = {
     CTF_HOST: process.env.CTF_HOST,
     CTF_SPACE_ID: process.env.CTF_SPACE_ID,
     CTF_ENVIRONMENT: process.env.CTF_ENVIRONMENT,
-    CTF_CDA_ACCESS_TOKEN: process.env.CTF_CDA_ACCESS_TOKEN, 
+    CTF_CDA_ACCESS_TOKEN: process.env.CTF_CDA_ACCESS_TOKEN,
```

Comment:
⭐️

Reply:
my editor does all the work for me 😉

Reply:
:) that would be great, i don't think there is such editor